setup.cfg

```diff
@@ -1,3 +1,6 @@
+[flake8]
+extend-ignore = E203,E501
```

Comment:
What's the deal with E203? :open_mouth: Also, see my comments on line length.

Reply:
Black does not follow those:

https://black.readthedocs.io/en/stable/faq.html#why-are-flake8-s-e203-and-w503-violated

Reply:
Just realized I forgot to add `W503` here too (according to link above)